pkgs/development/libraries/webkitgtk/2.18.nix

```diff
@@ -79,7 +79,7 @@ stdenv.mkDerivation rec {
 
   # XXX: WebKit2 missing include path for gst-plugins-base.
   # Filled: https://bugs.webkit.org/show_bug.cgi?id=148894
-  NIX_CFLAGS_COMPILE = "-I${gst-plugins-base.dev}/include/gstreamer-1.0"
+  NIX_CFLAGS_COMPILE = "-I${gst-plugins-base.dev}/include/gstreamer-1.0 -I${gst-plugins-bad}/include/gstreamer-1.0"
```

Comment:
This should not be necessary, the bug was already fixed: https://github.com/NixOS/nixpkgs/pull/29392/files#diff-5821c99a7cec601eb9518d9f22ed2727

Reply:
I am not removing it since it was already removed in https://github.com/NixOS/nixpkgs/commit/aebae6c20850ea1c29d2dad9d29b4db6c09e317c

Reply:
Yep, that should have rang a bell. I was just stupid. I'd even been thinking about this needing to go to staging, yet, it didn't occur to me, to look there.